src/server/typesMap.json

```diff
@@ -10,7 +10,7 @@
             "types": ["winjs"]
         },
         "Kendo": {
-            "match": "^(.*\\/kendo)\\/kendo\\.all\\.min\\.js$",
+            "match": "^(.*\\/kendo(-ui)?\\/kendo\\.all(\\.min)?\\.js$",

```

Comment:
Your parens are unbalanced. See the correct change you made in the file above.

Reply:
Good catch, fixed.